CMakeLists.txt

```diff
@@ -791,26 +812,31 @@ function(add_compiler_rt_tests variant)
     add_dependencies(check-llvm-toolchain-runtimes-${variant} check-compiler-rt-${variant})
 endfunction()
 
-function(add_libcxx_libcxxabi_libunwind_tests variant)
+function(add_libcxx_libcxxabi_libunwind_tests variant enable_exceptions enable_rtti)
+    set(target_name "")
+    extend_with_exception_and_rtti(target_name "libcxx_libcxxabi_libunwind_${variant}" ${enable_exceptions} ${enable_rtti})
+    set(target_postfix "")
+    extend_with_exception_and_rtti(target_postfix "${variant}" ${enable_exceptions} ${enable_rtti})
```

Comment:
Maybe `variant_with_exceptions`?

Reply:
I agree that in this use cases exception and rtti strings are applied after variant. But this function is not aware of that it simply applies necessary strings to the second parameter. In my opinion this name is adequate to what this function really does. Does this explanation convince you?

Reply:
Sorry. I meant `variant_with_extensions`.